docs/model/introduction-to-model-architecture.md

```diff
@@ -132,7 +132,7 @@ class CartRepository
         ]);
 
         $nativeQuery = $this->em->createNativeQuery(
-            'DELETE FROM carts WHERE modified_at <= :timeLimit AND user_id IS NULL',
+            'DELETE FROM carts WHERE modified_at <= :timeLimit AND user_id IS NOT NULL',
```

Comment:
are you sure with this, since user_id is nullable?

Reply:
Ye, sorry i copied the `deleteOldCartsForRegisteredCustomers()` instead. good point